constellations/gemini-user/src/internship/internship.js

```diff
@@ -0,0 +1,116 @@
+"use strict";
+require("./internship.sass");
+import React from "react";
+import ApCalcImgScr from "./../../assets/ap_calc.jpg";
+
+export class InternshipPage extends React.Component {
+    render() {
+        return (
+            <div id="view-intern">
+                <h1>Software Development Internship</h1>
+                <p>
+                    Math Navigator occassionally offers Software Development
+                    Internship for ambitious students who would like to pursuse
+                    a college major or a career in software development. In this
+                    internship oppotunity, students will learn the fundamentals
+                    of modern technologies commonly used across
+                    engineering-focused companies. And with these skills,
+                    students help create this site that you are currently using!
+                </p>
+                <h1>Technology Stack</h1>
+
+                <div class="container-main">
+                    <div class="child-image">
+                        <img class="image" src={ApCalcImgScr} />
+                    </div>
+                    <div class="child-caption">
+                        <p>
+                            Math Navigator occassionally offers Software
+                            Development Internship for ambitious students who
+                            would like to pursuse a college major or a career in
+                            software development.
+                        </p>
+                    </div>
+                </div>
+
+                <div class="container-main">
+                    <div class="child-image">
+                        <img class="image" src={ApCalcImgScr} />
+                    </div>
+                    <div class="child-caption">
+                        <p>
+                            Math Navigator occassionally offers Software
+                            Development Internship for ambitious students who
+                            would like to pursuse a college major or a career in
+                            software development.
+                        </p>
+                    </div>
```

Comment:
I think flex-items don't necessarily have to be `<div>` elements. They can very well be anything.
```
<div class="container-main">
    <img .../>
    <p>...</p>
</div>
```